Makefile

```diff
@@ -249,9 +249,9 @@ integration-test:
 # set to the build date/epoch. For more details, see: https://flit.pypa.io/en/latest/reproducible.html
 .PHONY: dist
 dist: dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-py3-none-any.whl dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION).tar.gz dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-docs-html.zip dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-build-epoch.txt
-dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-py3-none-any.whl: check test integration-test
+dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-py3-none-any.whl:
```

Comment:
Why are you removing `check test integration-test` from the `dist` target? The `dist` target should only run when the tests pass.

Reply:
Oops. I accidentally remove it. I will add it back.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/76/commits/f36424e2ba30eaf136e40616aa35fe71954fcc37